### ISSUE TITLE
[Bug Report] Issues with LG webOS

### ISSUE THREAD

**Issue by SirCumAlot1988** [stashapp/stash#4561]
**Describe the bug**
I have a LG TV at home. In general Stash works with the LG WebOS Browser and most importantly the scenes can be streamed without problems :-), but despite that I am having the following issues:

Scenes, Images and tags are loading fine. However, for movies, galleries, performers and studios only the first entries are loaded correctly (using the default 40 entries per page). If you scroll down, the images are missing for the following entries. Even if you wait for a few minutes, they are not loaded.

I attached a photo below.

The WebOS Browser is compatible with the magic remote. That's why I would like to use it ;-)

**To Reproduce**
1. Go to Movies/galleries/performers/studios
2. Scroll Down
3. Images are missing

**Expected behavior**
Images should be loaded correctly 

**Screenshots** 
![20240214_222042_modified](https://github.com/stashapp/stash/assets/98083850/b1e658d5-6355-44b0-a536-4210f2b6e0a6)

**Stash Version: (from Settings -> About):**
v0.24.3

**Desktop (please complete the following information):**
 - OS: LG WebOS 5.5.0-1103

**Comment by LeGrosFromage** [comment]
I happen to have a LG WebOS tv also so I gave this a try.  I set my "Scene" page to display 120 scenes.  I scrolled down, yes ones that were off-screen then scrolled on did indeed have the thumbnail missing - for about 2 seconds then they populated correctly.  The Studio logo was present the whole time.  So while I can reproduce @SirCumAlot1988's issue, mine corrects itself almost immediately.

Using latest release of Stash, not sure what version of WebOS but there are no updates pending for it.

**Comment by SirCumAlot1988** [comment]
> Have you tried different page sizes? Is it a specific number of images that are loaded/not loaded, or is it just the off-screen ones?
Yes, I tried different page sizes. It's just the off screen ones.



> I happen to have a LG WebOS tv also so I gave this a try. I set my "Scene" page to display 120 scenes. I scrolled down, yes ones that were off-screen then scrolled on did indeed have the thumbnail missing - for about 2 seconds then they populated correctly. The Studio logo was present the whole time. So while I can reproduce @SirCumAlot1988's issue, mine corrects itself almost immediately.
> 
> Using latest release of Stash, not sure what version of WebOS but there are no updates pending for it.
Yes, scenes work fine for me as well (see above). I am talking about movies, galleries, performers and studios. Could you try these on your side as well, please?

**Comment by LeGrosFromage** [comment]
Apologies, @SirCumAlot1988 - I missed where you'd called out scenes being ok - and of course, that was all I tried :-)

I just checked again and I have the same results as you.  Movies, Galleries, Performers and Studios do not display the thumbnails if their card was off-screen when the page is loaded.  Scrolling shows a broken thumbnail holder.  Scenes and Images do work as previously mentioned.  So yes 100% confirming your issue.

**Comment by GalacticCheat** [comment]
Maista sucks cock
![Image](https://github.com/user-attachments/assets/ed12369e-cb07-434a-b49f-fbc470272035)
https://i.ibb.co/ds49xcXt/image-2-copie.jpg

**Comment by feederbox826** [comment]
Tried loading emulator/ simulator but they don't provide browser directly :/

**Comment by feederbox826** [comment]
Unfortunately seems to be a issue with upstream WebOS not loading off-screen images. Closing as stale and probably upstream
